docs/resources/file.md.erb

```diff
@@ -43,7 +43,7 @@ content, size, basename, path, owner, group, type
 
 ### Unix/Linux Properties
 
-symlink, mode, link\_path, shallow\_link\_path, mtime, size, selinux\_label, md5sum, sha256sum, path, source, source\_path, uid, gid
+symlink, mode, link\_path, shallow\_link\_path, mtime, size, selinux\_label, md5sum, sha256sum, path, source, source\_path, uid, gid, more_permissive_than?
```

Comment:
`more_permissive_than?` is not a property (a value-fetching method, called using `its`), it is a matcher (a predicate).  It should be documented with the other matchers, at the end of the file.

Reply:
resolved in other PR